pageserver/src/layered_repository.rs

```diff
@@ -1611,8 +1611,15 @@ impl LayeredTimeline {
 
         let global_layer_map = GLOBAL_LAYER_MAP.read().unwrap();
         if let Some(oldest_layer) = global_layer_map.get(&layer_id) {
+            let last_lsn = self.get_last_record_lsn();
+            let n_delta_layers = if reconstruct_pages {
+                layers.count_delta_layers(oldest_layer.get_seg_tag(), last_lsn)
```

Comment:
Sorry, it is not my code so I do not want to rename variable I haven't introduce myself. But IMHO `oldest_layer` is better name, because it is obvious that `evict_layer` function is looking for layer to evict, `layer_to_evict` doesn't add any extra useful information. But   `oldest_layer` explains which layers is actually chosen for eviction and why.